client/src/Components/Alerts/AlertList.js

```diff
@@ -1,17 +1,38 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import webHost from '../../config/web';
+import { Redirect, useHistory} from 'react-router-dom';
 
-import Card from '../../UIElements/Card'
+import webHost from '../../config/web';
+import api from '../../config/web';
+import Card from '../../UIElements/Card';
+import Modal from '../../UIElements/Modal';
+import CreateAlert from './CreateAlert';
 
 const AlertList = () => {
   const [list, setList] = useState([]);
   const [loading, setLoading] = useState(true);
+  const history = useHistory();
 
+  const deleteHandler = (id) => {
+    //delete alert using the id
+    axios.delete(`${api.URI}/${id}`)
+      .then((response) => {
+        console.log(response.data);
+        //reload page after deleting alert
+        window.location.reload();
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
+
+  const updateHandler = (id) => {
```

Comment:
It seems like this update handler is not actually updating anything. Just adding the id to the route history

Reply:
If this is going to be tackled in another ticket, can you create a ticket in JIRA and add it here to the comments

``` 
// TODO (TRR-1234): add functionality for updating alert
const updateHandler = (id) => {
```

Reply:
Done. Changed the function to "update".

Reply:
This is actually not done. The name `updateHandler` wasn't the problem. The functionality is.

This function isn't actually updating the Alert object. It's just adding an ID to the history so the function isn't doing what you would expect it to based on the name

Reply:
My apologies. I believe I added that function to test an onClick button, but I didn't get it to work and didn't remove it.